ASM/c/get_items.c

```diff
@@ -424,8 +424,15 @@ void get_item(z64_actor_t *from_actor, z64_link_t *link, int8_t incoming_item_id
         row = get_item_row(item_id);
         if (row) {
             int16_t action_id = row->action_id;
-            if (CFG_ADULT_TRADE_SHUFFLE && action_id > 0 && from_actor->actor_id != 0x0A && IsAdultTradeItem(action_id)) {
-                if (action_id == Z64_ITEM_BIGGORON_SWORD) {
+            // Set adult trade item "traded" flags to prevent duping.
+            // Only necessary for full adult trade shuffle, except for Biggoron who always sets his flag.
+            if ((CFG_ADULT_TRADE_SHUFFLE || action_id == Z64_ITEM_BIGGORON_SWORD) // full adult trade shuffle on, or vanilla incoming item is Biggoron Sword
```

Comment:
The comments here are kind of messy. I'm open to clearer ways to write this.

Reply:
Seems fine to me.